test/hotspot/jtreg/vmTestbase/nsk/share/jdb/Launcher.java

```diff
@@ -137,6 +137,9 @@ private String[] makeJdbCmdLine (String classToExecute) {
                 /* Some tests need more carrier threads than the default provided. */
                 args.add("-R-Djdk.virtualThreadScheduler.parallelism=15");
             }
+            /* Some jdb tests need java.library.path setup for native libraries. */
+            String libpath = System.getProperty("java.library.path");
+            args.add("-R-Djava.library.path=" + libpath);
```

Comment:
Good, I wasn't sure how this inherit.

Reply:
This seems like an unrelated change. ???

Reply:
jdk.test.lib.thread.VThreadPinner has a native library that must be loaded.

Reply:
Ah I see. Thanks